### ISSUE TITLE
Comments in method chains

### ISSUE THREAD

**Issue by domaspoliakas** [tree-sitter/tree-sitter-scala#199]
### Commit of tree-sitter-scala you tested this on

7d348f51e442563f4ab2b6c3e136dac658649f93

### A code sample showing the error

```scala
def potato = 
  "hek"
    .tail
    // Hello
    .tail
    .tail
```

### Show the error node

```
function_definition [0, 0] - [6, 0]
  name: identifier [0, 4] - [0, 10]
  ERROR [1, 2] - [3, 4]
    field_expression [1, 2] - [2, 9]
      value: string [1, 2] - [1, 7]
      field: identifier [2, 5] - [2, 9]
  comment [3, 4] - [3, 12]
  body: indented_block [4, 4] - [6, 0]
    ERROR [4, 4] - [4, 5]
    field_expression [4, 5] - [5, 9]
      value: identifier [4, 5] - [4, 9]
      field: identifier [5, 5] - [5, 9]
```

### What do you expect the tree to look like

This is what it looks like if I take the comment out
```
function_definition [0, 0] - [5, 0]
  name: identifier [0, 4] - [0, 10]
  body: indented_block [1, 2] - [5, 0]
    field_expression [1, 2] - [4, 9]
      value: field_expression [1, 2] - [3, 9]
        value: field_expression [1, 2] - [2, 9]
          value: string [1, 2] - [1, 7]
          field: identifier [2, 5] - [2, 9]
        field: identifier [3, 5] - [3, 9]
      field: identifier [4, 5] - [4, 9]

```

### Where are you experiencing this error?

nvim-treesitter

**Comment by ghostbuster91** [comment]
I gave it a shot but I was unable to make the method chain with comment in the middle to parse correctly. 

Putting optional comment in the `field_expression` doesn't work. 

I found that it works for java parser https://github.com/tree-sitter/tree-sitter-java/blob/c194ee5e6ede5f26cf4799feead4a8f165dcf14d/grammar.js#L33 
I tried to modify scala grammar based on the java one but the issue remained.
